frontend/ui/elements/screensaver_menu.lua

```diff
@@ -153,12 +154,24 @@ return {
                         end,
                     },
                     {
-                        text = _("Stretch covers and images to fit screen"),
+                        text_func = function()
+                            if G_reader_settings:nilOrFalse("screensaver_stretch_images") then
+                                return _("Stretch to fit screen")
+                            elseif G_reader_settings:readSetting("screensaver_stretch_limit_percentage") then
+                                return T(_("Stretch to fit screen: limit to %1%"),
+                                    G_reader_settings:readSetting("screensaver_stretch_limit_percentage"))
+                            else
+                                return _("Stretch to fit screen: unlimited")
```

Comment:
"unlimited" sounds too wild :) it will be limited to the screen size.
Either nothing, or "no constraint", "no limit", "no restriction"...

Reply:
Maybe stretch to fit full sceen?